test/jdk/java/net/InetAddress/InternalNameServiceWithNoHostsFileTest.java

```diff
@@ -28,7 +28,8 @@
  *          a file name that contains address host mappings, similar to those in
  *          /etc/hosts file. TestHosts-II file doesn't exist, so a UHE should be
  *          thrown
- * @run main/othervm -Dsun.net.inetaddr.ttl=0  InternalNameServiceWithNoHostsFileTest
+ * @run main/othervm -Djdk.net.hosts.file=TestHosts-II -Dsun.net.inetaddr.ttl=0
+ *          InternalNameServiceWithNoHostsFileTest
```

Comment:
shouldn't this be:

```
@run main/othervm -Djdk.net.hosts.file=${test.src}/TestHosts-II -Dsun.net.inetaddr.ttl=0
```

Reply:
TestHosts-II was never in the test.src directory, from what I could tell it was just a dummy file (as per the test name) to verify the reaction when a Hosts File is not found. So, to be consistent with the other changes in this PR where a Host File is not present, I gave just the Filename in the path. Are there possible issues stemming from this? One I can think of is that a Hosts FIle of that name happens to be present in the user.dir though that is probably not very likely.

Reply:
Oh - OK - this is fine then